<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FuzzySync</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Browse1">
          <property name="text">
           <string>Browse1…</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Folder1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_addPair">
          <property name="text">
           <string>Add Pair</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit_Folder2"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Browse2">
          <property name="text">
           <string>Browse2…</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTreeWidget" name="treeWidget_FilesToCopy1">
        <column>
         <property name="text">
          <string>Files to Copy</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTreeWidget" name="treeWidget_FilesToCopy2">
        <column>
         <property name="text">
          <string>Files to Copy</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="treeWidget_FilesToOverwrite1">
        <column>
         <property name="text">
          <string>Files to Overwrite</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTreeWidget" name="treeWidget_FilesToOverwrite2">
        <column>
         <property name="text">
          <string>Files to Overwrite</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Copy1To2">
          <property name="text">
           <string>Copy 1 to 2</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_List">
          <property name="text">
           <string>List</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Abort">
          <property name="text">
           <string>Abort</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Synchronize">
          <property name="text">
           <string>Synchronize</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Copy2To1">
          <property name="text">
           <string>Copy 2 to 1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences…</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_Browse1</tabstop>
  <tabstop>lineEdit_Folder1</tabstop>
  <tabstop>pushButton_addPair</tabstop>
  <tabstop>lineEdit_Folder2</tabstop>
  <tabstop>pushButton_Browse2</tabstop>
  <tabstop>treeWidget_FilesToCopy1</tabstop>
  <tabstop>treeWidget_FilesToCopy2</tabstop>
  <tabstop>treeWidget_FilesToOverwrite1</tabstop>
  <tabstop>treeWidget_FilesToOverwrite2</tabstop>
  <tabstop>pushButton_Copy1To2</tabstop>
  <tabstop>pushButton_List</tabstop>
  <tabstop>pushButton_Abort</tabstop>
  <tabstop>pushButton_Synchronize</tabstop>
  <tabstop>pushButton_Copy2To1</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Browse1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse1_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Browse2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse2_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_List</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>list_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>647</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Copy1To2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copy1to2_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>652</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Synchronize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>synchronize_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Copy2To1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copy2to1_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_addPair</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addPair_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Abort</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>abort_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>750</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>774</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>browse1_clicked()</slot>
  <slot>browse2_clicked()</slot>
  <slot>list_clicked()</slot>
  <slot>copy1to2_clicked()</slot>
  <slot>synchronize_clicked()</slot>
  <slot>copy2to1_clicked()</slot>
  <slot>addPair_clicked()</slot>
  <slot>abort_clicked()</slot>
 </slots>
</ui>
